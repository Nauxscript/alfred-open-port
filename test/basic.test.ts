import { describe, expect, it } from 'vitest'
import run from '../src/index'

describe('index entry', () => {
  it('happy path', () => {
    expect(1 + 1).toEqual(2)
  })

  it('query string is empty', () => {
    const result = run(['', true])
    expect(result).toMatchSnapshot()
  })
})
