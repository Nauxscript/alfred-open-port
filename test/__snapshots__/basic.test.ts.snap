// Vitest Snapshot v1

exports[`index entry > query string is empty 1`] = `
{
  "items": [
    {
      "title": "This is a selection example, and you are inputing ",
    },
  ],
}
`;
